src/redux/sagas/actions/unlockToken.ts

```diff
@@ -100,6 +98,8 @@ function* tokenUnlockAction({
       meta,
     });
 
+    setTxHash?.(txHash);
```

Comment:
I wonder if this shouldn't be moved down in every saga to make sure we will get complete data from `getColonyAction` right after creating the action.

Also, I have an issue in the local environment where `createColonyActionMetadata` mutation would get stuck in a pending state and never reach `setTxHash?.(txHash);` call. Do you know if this could be related somehow to payloads sent from forms?

![image](https://github.com/JoinColony/colonyCDapp/assets/38806520/5830c299-d9c2-4e3f-a953-d10655f92ea7)
![image](https://github.com/JoinColony/colonyCDapp/assets/38806520/5b06247f-6953-4f1c-ba89-1d3b2e3bfe63)